core/spec.md

```diff
@@ -407,7 +407,7 @@ attributes.
 - **Resource**
 
 Resources, typically, represent the main data of interest for the Registry. In
-the filesystem analogy, these would be the "files". All Resources exist under
+the filesystem analogy, these would be the "files". Each Resources exist under
```

Comment:
Sorry for the added nit: "Each Resource exists"

Reply:
fixed